internal/subscription/model_test.go

```diff
@@ -62,6 +62,7 @@ func Test_Model(t *testing.T) {
 	assert.False(t, dSubscription.IsPush())
 	assert.Nil(t, dSubscription.GetCredentials())
 	assert.False(t, dSubscription.HasCredentials())
+	assert.Empty(t, dSubscription.GetRedactedPushEndpoint())
```

Comment:
Can you add a test case for redacted pushendpoint ?
eg: redacted push endpoint != push endpoint if password is there 
       redacted push endpoint == push endpoint if there is no secret in the url

Reply:
Sure